flight/utils/movement_controller.py

```diff
@@ -95,25 +96,52 @@ async def move_to(self, drone: System, pylon: LatLon) -> bool:
                 return True
             count += 1
 
-    async def turn(self, drone: System) -> bool:
+    async def turn(self, drone: System, degrees_turned = 180: int, left_turn = True: bool, time = 3.5: float) -> bool:
```

Comment:
Another possible issue I noticed is that adding these arguments is not reflected when the function is called. That is, in `states/early_laps.py`, the call to `mover.turn()` is only passed a `drone`. Are we planning on using the default arguments for the pylon laps?